docker/elastic.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.3'
+# Elastic config, to be appended after base-prod or base-dev, ..., but before selenium.yml
+#
+# NOTE: You'll need to manually run the command: php bin/console ibexa:elasticsearch:put-index-template.
+
+## WARNING!
+# This service is currently work in progress, is not tested by CI, and thus not guaranteed to work.
+# You are however more then welcome to try it out and help make it stable.
+
+services:
+  app:
+    depends_on:
+      - elasticsearch
+    environment:
+      - SEARCH_ENGINE=elasticsearch
+      - ELASTICSEARCH_DSN=elasticsearch:9200
+
+  elasticsearch:
+    build:
+      context: ../../
+      dockerfile: doc/docker/Dockerfile-elastic
```

Comment:
This works fine, but I think we could just use the `image` property.

Examples:
https://github.com/ibexa/docker/blob/main/docker/percy.yml#L6
https://github.com/ibexa/docker/blob/main/docker/selenium.yml#L6

This way we could get rid of the Dockerfile - because there are no custom steps, specifying an image should be enough